geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseIndexAdapter.scala

```diff
@@ -87,17 +87,26 @@ class HBaseIndexAdapter(ds: HBaseDataStore) extends IndexAdapter[HBaseDataStore]
 
         val cols = groups.apply(index.sft).map(_._1)
         val bloom = Some(BloomType.NONE)
-        val encoding = if (index.name == IdIndex.name) { None } else { Some(DataBlockEncoding.FAST_DIFF) }
+        val encoding = if (index.name == IdIndex.name) {
```

Comment:
i mean all these changes. i think pretty much everything above line 606 can be reverted.

Reply:
Yeah, I was manually fixing tabs, but I'll just go into the latest commit from main and copypaste to overwrite.

Reply:
in intellij you can do a right-click->git->compare with branch->main, that will bring up a diff view and let you review and copy over each change.